src/components/Simulering/Simulering.tsx

```diff
@@ -276,6 +277,22 @@ export const Simulering = ({
           detaljer={detaljer}
         />
       )}
+
+      {!(
+        isOffentligTpLoading ||
+        isLoading ||
+        isPensjonsavtalerLoading ||
+        isEndring
+      ) &&
```

Comment:
Kanskje en personlig preferanse, men jeg synes dette er enklere å lese:
```suggestion
      {!isOffentligTpLoading &&
        !isLoading &&
        !isPensjonsavtalerLoading &&
        !isEndring &&
```